Vendor/iOS/MSAlertController/MSAlertController.m

```diff
@@ -146,4 +146,25 @@ + (void)presentNextAlertAnimated:(BOOL)animated {
   }
 }
 
+#define DISPATCH_SELECTOR(class, selectorName, ...) ({ \
+  SEL selector = NSSelectorFromString(@#selectorName); \
+  IMP impl = [class methodForSelector:selector]; \
+  ((id (*)(id, SEL, ...)) impl)(class, selector, ##__VA_ARGS__); \
+})
+
++ (void)makeKeyAndVisible {
+  if (@available(iOS 13.0, *)) {
+    UIApplication *application = (UIApplication *)DISPATCH_SELECTOR([UIApplication class], sharedApplication);
+    NSSet *scenes = (NSSet *)DISPATCH_SELECTOR(application, connectedScenes);
+    for (NSObject *scene in scenes) {
+      NSInteger activationState = (NSInteger)DISPATCH_SELECTOR(scene, activationState);
+      if (activationState == 0 /* UISceneActivationStateForegroundActive */) {
```

Comment:
The issue I see here is when there is no active scenes to begin with. For instance in Distribute the alert can pop up anytime, likely at the beginning of the user session but it's triggered by an HTTP call result so it's async. At that time all of the scenes can be hidden by, let's say, the drop down notification center, in this case all scenes are inactive. We need to find a way to cover this case too.